lib/storage/file.js

```diff
@@ -1153,11 +1153,52 @@ File.prototype.getSignedPolicy = function(options, callback) {
  * @param {string} callback.url - The signed URL.
  *
  * @example
+ * var request = require('request');
+ * var TWO_WEEKS_MS = Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14);
+ *
+ * //-
+ * // Generate a URL that allows temporary access to download your file.
+ * //-
  * file.getSignedUrl({
```

Comment:
Just a nitpick that you can feel free to ignore :)

Would it be better to require your modules within this code block? I feel like it might look a little funny to start off our example with a code block that just creates 2 variables.